src/app/space/create/codebases/codebases-item-actions/codebases-item-actions.component.ts

```diff
@@ -1,14 +1,13 @@
-import {Component, Input, OnDestroy, OnInit, ViewChild, ViewEncapsulation} from '@angular/core';
+import { Component, Input, OnDestroy, OnInit, ViewChild, ViewEncapsulation } from '@angular/core';
 import { Subscription } from 'rxjs';
 
-import { Che } from '../services/che';
 import { Codebase } from '../services/codebase';
 import { CodebasesService } from '../services/codebases.service';
 import { Broadcaster, Notification, NotificationType, Notifications } from 'ngx-base';
 import { Dialog } from 'ngx-widgets';
 import { WindowService } from '../services/window.service';
 import { WorkspacesService } from '../services/workspaces.service';
-import {IModalHost} from "../../../wizard/models/modal-host";
+import { IModalHost } from '../../../wizard/models/modal-host';
 
 @Component({
   encapsulation: ViewEncapsulation.None,
```

Comment:
There is an unused cheRunning input in this class that should be removed.

https://github.com/ashumilova/fabric8-ui/blob/4a59ebc3452e43902418fc118a462a4c98e39a4f/src/app/space/create/codebases/codebases-item-actions/codebases-item-actions.component.ts#L19

Reply:
used in template https://github.com/ashumilova/fabric8-ui/blob/4a59ebc3452e43902418fc118a462a4c98e39a4f/src/app/space/create/codebases/codebases-item-actions/codebases-item-actions.component.html#L7